src/integrators/controllers.jl

```diff
@@ -132,8 +132,8 @@ end
   if iszero(EEst)
     q = inv(qmax)
   else
-    q11 = DiffEqBase.fastpow(EEst, beta1)
-    q = DiffEqBase.value(q11 / DiffEqBase.fastpow(qold, beta2))
+    q11 = DiffEqBase.fastpow(DiffEqBase.value(EEst), float(beta1))
```

Comment:
@ChrisRackauckas are the remaining `DiffEqBase.value(EEst)` and  `DiffEqBase.value(qold)` ok?
Or do they need to be set somewhere else? E.g. in https://github.com/SciML/OrdinaryDiffEq.jl/blob/e50ebf5c2c7b9ee8bdf079919add78851796d8f5/src/solve.jl#L394

Reply:
Just never convert to `QT` and then `DiffEqBase.value` on those won't be necessary.